lib/checks/keyboard/no-focusable-content-evaluate.js

```diff
@@ -41,8 +41,7 @@ function getFocusableDescendants(vNode) {
 
   const retVal = [];
   vNode.children.forEach(child => {
-    const role = getRole(child);
-    if (getRoleType(role) === 'widget' && isFocusable(child)) {
+    if (getRoleType(child) === 'widget' && isFocusable(child)) {
```

Comment:
If you want me to pull these into their own PR let me know. It's not unrelated to the PR... but it's not very related either.